src/components/Operations/Operation.jsx

```diff
@@ -26,7 +26,9 @@ const capitalizeFirstLetter = str => {
 export const Operation = ({ operation }) => {
   const client = useClient()
 
-  const [category, setCategory] = useState(operation.cozyCategoryId || '')
+  const [category, setCategory] = useState(
+    operation.cozyCategoryId || operation.manualCategoryId || ''
```

Comment:
In banks, the `manualCategoryId` is the source of truth, it is more: `operation.manualCategoryId || operation.cozyCategoryId || ''`